pkg/kubelet/config/flags.go

```diff
@@ -33,12 +33,12 @@ type ContainerRuntimeOptions struct {
 	PodSandboxImage string
 	// Image credential provider plugin options
 
-	// ImageCredentialProviderConfigFile is the path to the credential provider plugin config file.
+	// ImageCredentialProviderConfigPath is the path to the credential provider plugin config file.
 	// This config file is a specification for what credential providers are enabled and invoked
 	// by the kubelet. The plugin config should contain information about what plugin binary
 	// to execute and what container images the plugin should be called for.
```

Comment:
update the godoc to indicate this can be a file or a directory, and if a directory, all .json/.yaml/.yml files in the directory are loaded and merged in lexicographical order

Reply:
Done